Gemfile

```diff
@@ -3,7 +3,7 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.4.0'
 
-
+gem 'sqlite3', '~> 1.3.6'
 gem 'rubocop', '~> 0.67.2'
```

Comment:
Marcus, you need to take sqlite3 out of this section.  Because we are pushing to heroku, we can't require sqlite3 in production.  We can only require it in development and test -- and you will see below that it is included for those.